### ISSUE TITLE
Can you give me the details of the crowd count procedure? Thank you

### ISSUE THREAD

**Issue by 1691077362** [golmschenk/sr-gan#6]
Can you give me the details of the crowd count procedure? Thank you

**Comment by golmschenk** [comment]
Hello! The explanation of the crowd counting method can be found in [the corresponding paper](https://www.sciencedirect.com/science/article/pii/S1077314219300955). To run the crowd experiment, set the application to the crowd application. This is done in `run.py`, near the top. Change the line for `application_name` to be `application_name = ApplicationName.crowd`.

**Comment by 1691077362** [comment]
Thank you

**Comment by 1691077362** [comment]
Should I run the utility.py to get the crowd count results

**Comment by 1691077362** [comment]
Should I run the utility.py to get the crowd count results？ @golmschenk

**Comment by golmschenk** [comment]
@1691077362, during training, train and validation metrics, along with various other information, is logged to the `logs` directory in a TensorBoard format. To view these results, run [TensorBoard](https://www.tensorflow.org/tensorboard/get_started) on the logs directory. To run the final test evaluation, in `run.py`, change `experiment.train()` to `experiment.evaluate()`.

**Comment by 1691077362** [comment]
@golmschenk     File "E:\RQJS\sr-gan-master\crowd\srgan.py", line 307, in evaluate
                                  test_dataset = self.settings.dataset_class(dataset='test')
                           AttributeError: 'Settings' object has no attribute 'dataset_class'
Does the Dataset () in this Settings need to be written？

**Comment by 1691077362** [comment]
@golmschenk  File "E:\RQJS\sr-gan-master\crowd\srgan.py", line 307, in evaluate
    test_dataset = self.settings.dataset_class(dataset='test')
AttributeError: 'Settings' object has no attribute 'dataset_class'
Does the dataset_class（） in this Settings need to be written

**Comment by golmschenk** [comment]
It appears I did not have `evaluate()` automatically setup the dataset. Just to note, the `train()` script also runs a version of the evaluation to get performance metrics, in case that's sufficient.

If not, running `experiment.dataset_setup()` before `experiment.evaluate()` should work for most of the datasets. Though, it looks like I did not have all the possible datasets in the `dataset_setup` method, so depending on which one you are using, it might require additional setup.
